docs/source/diagnostics.rst

```diff
@@ -0,0 +1,86 @@
+Diagnostics
+===========
+
+Profiling parallel code can be tricky. ``dask.diagnostics`` provides
+functionality to aid in profiling and inspecting dask graph execution.
+
+
+Scheduler Callbacks
+-------------------
+
+Schedulers based on ``dask.async.get_async`` (currently
+``dask.async.get_sync``, ``dask.threaded.get``, and
+``dask.multiprocessing.get``) accept two callbacks, allowing for inspection of
+dask execution. The callbacks are:
+
+1. ``start_callback(self, key, dask, state)``
```

Comment:
Probably want some sort of divider here, maybe a colon `:` or an extra empty line.